src/cf-layout/src/molecules/heroes.less

```diff
@@ -62,133 +66,121 @@
         } );
     }
 
-    &_subhead,
-    &_cta:not( .btn ) {
+    &_subhead {
         // Not using the `.heading-3()` mixin here because we want the weight
         // to remain Regular on smaller screens.
         .u-webfont-regular();
 
-        font-size: @size-iv;
+        font-size: @size-iii;
         line-height: 1.25;
 
-        .respond-to-min( @bp-med-min, {
-            font-size: @size-iii;
-        } );
-    }
-
-    &_subhead:last-child {
-        margin-bottom: 0;
-    }
-
-    &_cta {
-        &:not( .btn ) {
-            border-bottom-width: 1px;
-        }
-
-        .respond-to-min( @bp-sm-min, {
-            margin-bottom: 0;
+        .respond-to-max( @bp-sm-max, {
+            font-size: @size-iv;
         } );
     }
 
     &_image-wrapper {
         .grid_column( 1, 1 );
 
+        overflow: hidden;
+
         .respond-to-min( @bp-sm-min, {
             .grid_column( 5, 12 );
 
-            // Keep the image vertically centered
+            margin: auto;
+
+            // IE specific fix
             display: table-cell;
             vertical-align: middle;
+
+            @supports ( display: flex ) {
+                display: block;
+            }
         } );
 
         .respond-to-max( @bp-xs-max, {
             margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
         } );
+    }
 
-        &__bleed-vertical {
-            position: relative;
+    &_image {
+        .u-flexible-container-mixin( 16, 9 );
 
-            .m-hero_image {
-                display: block;
-                height: auto;
-            }
+        background-position: center;
+        background-repeat: no-repeat;
+        background-size: contain;
+    }
 
-            .m-hero_image__bleeding {
-                width: 100%;
-                height: 100%;
-                position: absolute;
-                top: 0;
-                background-position: center;
-                background-repeat: no-repeat;
-                background-size: cover;
+    &__knockout {
+        background-color: @hero-knockout-bg;
+        color: @hero-knockout-text;
+    }
+
+    &__bleeding {
+        .respond-to-min( @bp-sm-min, {
+
+            .m-hero_wrapper {
+                position: relative;
             }
 
-            .respond-to-max( @bp-xs-max, {
-                .m-hero_image__bleeding {
-                    display: none;
-                }
-            } );
+            .m-hero_image-wrapper {
+                margin-top: unit( @grid_gutter-width / @base-font-size-px, em ) * -1;
+                margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em ) * -1;
 
-            .respond-to-min( @bp-sm-min, {
-                .m-hero_image {
-                    display: none;
-                }
-            } );
+                //IE specific fix
```

Comment:
IE10 and 11 do, like you said, with bugs (and it's serious bugs) but they don't report that they do through `@supports`, so the table cell alignment is for all IE browsers and whatever webkit ones that don't report support for flex. As of now, the latest builds of Chrome, Safari, and FF are the only ones reporting through `@supports` as actually supporting Flex